src/components/Slider.js

```diff
@@ -33,7 +41,7 @@ const Slider = () => {
       <Swiper {...params}>
         {products.map(({ img1, title, price, id }) => (
           <div className="single-product" key={id}>
-            <Link to={`/item/${id}`}>
+            <Link to={`/item/${id}`} onClick={productCountReSetHandler(id)}>
```

Comment:
why we should reset this? I am clicking to see product detail, there is no need to reset the product count in my card.